dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -390,7 +390,7 @@
 		status = "disabled";
 		prescaler = <2>;
 		label = "SC_TIMER";
-		#pwm-cells = <2>;
+		#pwm-cells = <3>;
```

Comment:
The same should be done in nxp_rt5xx_common.dtsi:
https://github.com/zephyrproject-rtos/zephyr/blob/536fe6a948c9a546bb89dc73ab4adbd3b13b3e71/dts/arm/nxp/nxp_rt5xx_common.dtsi#L350
and in nxp_rt6xx_common.dtsi:
https://github.com/zephyrproject-rtos/zephyr/blob/536fe6a948c9a546bb89dc73ab4adbd3b13b3e71/dts/arm/nxp/nxp_rt6xx_common.dtsi#L302

Reply:
fixed